extension/content.js

```diff
@@ -294,6 +294,30 @@ function markMergeCommitsInList() {
 	});
 }
 
+function indentInput(el, size = 4) {
+	el.focus();
+	const val = el.value;
+	const selectionStart = el.selectionStart;
+	const indentSize = (size - el.selectionEnd % size) || size;
+	const indentationText = ' '.repeat(indentSize);
+	el.value = val.slice(0, selectionStart) + indentationText + val.slice(el.selectionEnd);
```

Comment:
I'm sorry for seeing this later, but I think `value` is better than `val`. What do you think?